packages/webapp/pages/devcards/[userId].tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import {
+  DevCard,
+  DevCardType,
+} from '@dailydotdev/shared/src/components/profile/devcard';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+
+export async function getStaticPaths(): Promise<GetStaticPathsResult> {
+  return { paths: [], fallback: 'blocking' };
+}
+
+interface DevCardPageProps {
+  userId: string;
+}
+
+export async function getStaticProps({
+  params,
+}: GetStaticPropsContext): Promise<GetStaticPropsResult<DevCardPageProps>> {
+  const { userId } = params;
+
+  if (!userId) {
+    return {
+      notFound: true,
+      revalidate: 60,
+    };
+  }
+
+  return {
+    props: { userId: userId as string },
+    revalidate: 60,
+  };
+}
+
+const DevCardPage = ({ userId }: DevCardPageProps): ReactElement => {
+  return (
+    <div id="screenshot_wrapper" className="w-[356px]">
+      <DevCard userId={userId} type={DevCardType.Vertical} />
```

Comment:
I think the API introduces a type property as part of the query params. Should we utilize it here and have the `DevCardType.Vertical` as fallback?

Reply:
Right now we only support the vertical type. The API is not sending query params at the moment. That's an improvement waiting to happen :)

Reply:
Fixed 👍